cmd/admin-revoker/main.go

```diff
@@ -210,6 +213,48 @@ func (r *revoker) revokeBySerialBatch(ctx context.Context, serialPath string, re
 	return nil
 }
 
+func (r *revoker) revokeIncidentTableSerials(ctx context.Context, tableName string, reasonCode revocation.Reason, parallelism int) error {
+	wg := new(sync.WaitGroup)
+	work := make(chan string, parallelism)
+	for i := 0; i < parallelism; i++ {
+		wg.Add(1)
+		go func() {
+			defer wg.Done()
+			for serial := range work {
+				err := r.revokeBySerial(ctx, serial, reasonCode, false)
+				if err != nil {
+					r.log.Errf("failed to revoke %q: %s", serial, err)
+				}
+			}
+		}()
+	}
+
+	stream, err := r.sac.SerialsForIncident(ctx, &sapb.SerialsForIncidentRequest{IncidentTable: tableName})
+	if err != nil {
+		return fmt.Errorf("setting up stream of serials from incident table %q: %s", tableName, err)
+	}
+
+	var atLeastOne bool
+	for {
+		is, err := stream.Recv()
+		if err != nil {
+			if err == io.EOF {
+				if !atLeastOne {
+					r.log.AuditInfof("No serials found in incident table %q", tableName)
+				}
```

Comment:
This is a very minor thing, but I would do this outside the loop. It reduces the amount of nesting here, and simplifies the code to to a very recognizable pattern:
```
set sentinel
for {
  maybe update sentinel
}
check sentinel
```